journal-temp/modernized/15-chapter-15.adoc

```diff
@@ -1065,10 +1141,10 @@ Blessed be the Lord, they were disappointed,
 that that weapon was not formed ready to be put into their hands,
 or power to make use of.
 
-Exceptions against the Bill, entitled An Act for the better payment of Church-rates,
-Small Tithes, and other Church-duties; and for better passing Church Wardens accounts;
-humbly offered.
+[.blurb]
+=== Exceptions against the Bill, entitled An Act for the better payment of Church-rates, Small Tithes, and other Church-duties; and for better passing Church Wardens accounts; humbly offered.
```

Comment:
I wasn't sure if this should be embedded. It was in the ODT. I quickly read through it and it didnt feel right embedded because it didn't seem like a real document, but something he was talking about/explaining. But I'll leave it up to you!